recurly/__init__.py

```diff
@@ -959,6 +960,8 @@ class Invoice(Resource):
         'billing_info',
         'billing_info_uuid',
         'dunning_campaign_id',
+        'unit_amount_in_cents',
```

Comment:
Shouldn't only the adjustment have `unit_amount_in_cents`?

Reply:
you're right! changing